### DISCUSSION TITLE
Installation - boot partition

### DISCUSSION THREAD

**Discussion by GeneralAccess** [elementary/installer#733] [Q&A] [answered]
I tried ElementaryOS several years ago, but have been using other distributions since. I'd like to try and possibly switch to elementaryOS as I have always admired its elegance. With the installers I am used to, I create a root partition '/' and a home partition '/home' and there is a tick box for identifying the boot partition, I choose '/'.

The elementaryOS installer seems to be asking for a third, boot, partition. I am confused.

It also recommends a swap partition, I could add that, I think there is a rule-of-thumb of twice the RAM size.
The hardware requirements state SSD of n GB. I have assumed that this is only a recommendation and a HDD will do.

**Comment by lenemter** [comment, marked as answer]
I think you need to create a third patrition about 500mb in size and select it as /boot/efi

**Reply by GeneralAccess** [reply]
That worked. I like the fact that the installer takes you into GParted (which I'm familiar with) to set up the partitions. The installation was easy and quick after that, and it didn't get into a tangle when I selected the BTRFS disk format.
It's good that Flatpaks are mentioned straight away rather than having to be discovered, although I've no idea what "Sideloading" is. Installing a Flatpak was a bit awkward as I tried installing Sideload (is that what Sideloading means?) and the button changed to "Open" but nothing happened. Eventually found it.
I'll wait and see how the next release of my current distribution looks, then it's possible I might switch to elementaryOS. That would require unpatronising support (OK so far :)) and integration of the apps I currently use (most should be OK as I would have the option of using the Flatpak instead of what is bundled with the OS).
Thanks for your advice.
